src/backend/replication/walproposer.c

```diff
@@ -1953,6 +1953,9 @@ WalproposerShmemInit(void)
 	if (!found)
 	{
 		memset(walprop_shared, 0, WalproposerShmemSize());
+		walprop_shared->feedback.ps_writelsn = UnknownXLogRecPtr;
```

Comment:
This part looks sane to me.
It's natural to init this fields with "unknown" value, so that we didn't use them before receiving any real feedback from pageserver.

A comment wouldn't hurt, though)